doc/classes/ColorPicker.xml

```diff
@@ -19,19 +19,40 @@
 				[b]Note:[/b] The presets list is only for [i]this[/i] color picker.
 			</description>
 		</method>
+		<method name="add_recent_preset">
+			<return type="void" />
+			<param index="0" name="color" type="Color" />
+			<description>
+				Adds the given color to a list of color recent presets. The recent presets are displayed in the color picker and the user will be able to select them.
+				[b]Note:[/b] The recent presets list is only for [i]this[/i] color picker.
```

Comment:
Maybe you could better clarify what are the recent presets and the purpose of this functions. Like, recent presets are colors that were used recently and a new preset is automatically created when you pick a new color. The functions for manipulating recent presets can be used to store them for later use or to use between multiple color pickers etc.